# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../hotplug2/modwrap-fallback.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./linux24_compat/hotplug2-modwrap.c	(revision 100)
+++ ./linux24_compat/hotplug2-modwrap.c	(working copy)
@@ -122,6 +122,12 @@
 		free(module);
 		free(line);
 	}
+
+	if (strcmp(argv[argc - 1], match_alias) == 0) {
+		if (execute(argv)) {
+			ERROR("execute", "Unable to execute: `%s'.", argv[0]);
+		}
+	}	
 	
 	free(filename);
 	free(match_alias);
